src/components/Dropdowns/SelectionDropdown.tsx

```diff
@@ -75,29 +98,92 @@ export default function SelectionDropdown(inputProps: SelectionDropdownProps): R
     return "";
   }, [props.selected, items]);
 
+  // Set up fuse for fuzzy searching
+  const fuse = useMemo(() => {
+    return new Fuse(items, {
+      keys: ["key", "label"],
+      isCaseSensitive: false,
+      shouldSort: true,
```

Comment:
sorts alphabetically?  is that only on typed search results?

Reply:
`shouldSort` sorts by the matching score, i.e. how close the match is to the search string. `isCaseSensitive` and `shouldSort` are set to these values by default, but I originally explicitly set their values so it would be more obvious that that behavior was taking place. I can probably just remove them.

Reply:
Looks fine to me, just wanted to understand what it actually means :)